views/index.handlebars

```diff
@@ -1,6 +1,5 @@
-<div id="react-nav-bar"></div>
 <div class="row row--tall flex-container--column">
-    <div id="react-publish-tool" class="row row--padded row--tall flex-container--column">
+    <div id="react-app" class="row row--tall flex-container--column">
         <div class="row row--padded row--tall flex-container--column flex-container--center-center">
             <p>loading...</p>
```

Comment:
I meant to keep the loading bar in so it will render until the app is rendered.  Once the app renders it replaces the content of the div and removes the loading bar.